src/renderer/controllers/torrent-list-controller.js

```diff
@@ -40,6 +40,11 @@ module.exports = class TorrentListController {
 
   // Shows the Create Torrent page with options to seed a given file or folder
   showCreateTorrent (files) {
+    // You can only create torrents from the home screen.
+    if (this.state.location.url() !== 'home') {
+      return dispatch('error', 'Please go back to the torrent list before creating a new torrent.')
```

Comment:
It's not ideal that this error appears in two places. Can't it be removed from here? With the changes in this PR, it's not possible for the drag-and-drop handler to call `showCreateTorrent` except from the home screen anyway.